include/zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld

```diff
@@ -329,6 +301,33 @@ SECTIONS
 
     __data_region_end = .;
 
+    SECTION_DATA_PROLOGUE(_BSS_SECTION_NAME,(NOLOAD), BSS_ALIGN)
+    {
+        /*
+         * For performance, BSS section is assumed to be 4 byte aligned and
+         * a multiple of 4 bytes
+         */
+        . = ALIGN(4);
+        __bss_start = .;
+        __kernel_ram_start = .;
```

Comment:
In this way, should `__kernel_ram_start` be put above the `.data` section? Otherwise, `__kernel_ram_start` would not include the data section, right?

Reply:
Let me check where this marker is used, you likely have a point there. This was missed by the author of #53262 as well.

Also, I noticed last night that with regards to the linker command file, the issue described in #51024 also still persists. We should preferrably fix this as well before going for the A/R/M separation.